src/web/assets/cp/src/js/BaseElementIndex.js

```diff
@@ -861,7 +869,11 @@ Craft.BaseElementIndex = Garnish.Base.extend(
       return params;
     },
 
-    updateElements: function (preservePagination, pageChanged) {
+    updateElements: function ({
+      preservePagination,
+      pageChanged,
+      sortUpdated,
+    } = {}) {
```

Comment:
@brandonkelly wondering if this is the way you would approach this. I had also considered adding additional events, like `sortUpdated`, `sourceChanged`, but this seemed like it might be more straightforward.

Reply:
We can’t make a backwards-incompatible change to `updateElements` like this, as some plugins may be calling it themselves.

We could add the additional parameter, but it seems like would just start announcing the message directly from the 3 different places you are passing `sortUpdated: true`?

Reply:
That makes sense. So instead of bundling this functionality into the `updateElements` function, just calling the `_updateScreenReaderStatus` function directly after `updateElements`?

I can give that a try!